app/assets/stylesheets/pages/_package_index.scss

```diff
@@ -1,33 +1,86 @@
-.package-cards {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  height: 65vh;
-}
+  .package-cards {
+  height: 800px;
+  margin: auto;
+  display: grid;
+  // justify-items: center;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 30px;
+  align-content: stretch;
+  justify-items: center;
+  }
 
-.package-card {
-  height: 35%;
-  width: 300px;
-  margin-bottom: 20px;
-  margin-top: 20px;
-  padding-top: 30px;
-  padding-bottom: 30px;
-}
+  .package-card {
+    height: 45%;
+    width: 500px;
+    // margin-bottom: auto;
+    // margin-left: 80px;
+    padding-top: 30px;
+    padding-bottom: 30px;
+    background-size: cover;
+  }
+
+  .container-package {
+    margin-right: auto;
+  }
+
+  .package-info {
+    text-align: center;
+    background-color: rgba(214, 212, 208, 0.6);
+    height: 50%;
+    // margin-top: 15px;
+    border-radius: 5px;
+    }
+
+   .package-info h3 {
+      margin: 0;
+      font-size: 30px;
+      background-color: #0FABBC;
```

Comment:
Try to use the colorvarriables, this makes it more clear. If there is no variable for this particualr colour yet, use this link to get a suitable name: http://chir.ag/projects/name-that-color/#BCD7BC